pepper-apis/parent-pom.xml

```diff
@@ -451,6 +451,11 @@
             <artifactId>redisson</artifactId>
             <version>3.13.4</version>
         </dependency>
+        <dependency>
+            <groupId>redis.clients</groupId>
+            <artifactId>jedis</artifactId>
+            <version>3.3.0</version>
+        </dependency>
```

Comment:
Completely separate library to talk to Redis. Just to eliminate possibility that using the same library (Redisson) could mess up our test. Can go away if we verify that connectivity is not a problem